### ISSUE TITLE
'mc.cores' > 1 is not supported on Windows

### ISSUE THREAD

**Issue by cfz1998** [kharchenkolab/pagoda2#141]
Hi! Everyone.
I flowing the "Pagoda2 Walkthrough" step-by-step. But i got a error in 
![image](https://user-images.githubusercontent.com/60096719/210752732-af82b8dc-2c53-4084-9fcf-d4eab6eb8a0d.png)
How to solve this problem?
Thank you for your reply!
--zcf

**Comment by evanbiederstedt** [comment]
Hi there

This is actually an error within the base R `parallel` package for `mclapply`: 

https://github.com/wch/r-source/blob/f745892151035f273af88e24c943f5549abb42e9/src/library/parallel/R/windows/mcdummies.R#L19-L27

`mclapply` doesn't work on Windows, as explained here: https://www.r-bloggers.com/2014/07/implementing-mclapply-on-windows-a-primer-on-embarrassingly-parallel-computation-on-multicore-systems-with-r/

Note that there's another issue here whereby the error was elsewhere: https://github.com/kharchenkolab/pagoda2/issues/120

In summary:

If there is only one core set, the function should work, i.e. 

```
r <- Pagoda2$new(counts, log.scale=TRUE, n.cores=1)
```

I could doublecheck on Windows in the future at some point.

Best, Evan

**Comment by cfz1998** [comment]
> 

OHH! Thank you!
